backend/webapi/Features/ClientLogs/Create.cs

```diff
@@ -0,0 +1,50 @@
+namespace Pidp.Features.ClientLogs;
+
+using FluentValidation;
+using Pidp.Data;
+using Pidp.Models;
+
+public class Create
+{
+    public class Command : ICommand
+    {
+        public string Message { get; set; } = string.Empty;
+
+        public LogLevel? LogType { get; set; }
+
+        public string BrowserInformation { get; set; } = string.Empty;
+    }
+
+    public class CommandValidator : AbstractValidator<Command>
+    {
+        public CommandValidator(IHttpContextAccessor accessor)
+        {
+            var user = accessor?.HttpContext?.User;
```

Comment:
You don't need to do any validation against the `User`, so you don't need this line or to inject the `IHttpContextAccessor`.